src/levels.rs

```diff
@@ -519,6 +519,14 @@ impl SegmentId {
         // Cheaply make sure it is in bounds in a way the compiler can see at call sites.
         self.id % RAV1D_MAX_SEGMENTS
     }
+
+    pub fn min() -> Self {
+        Self::new(0).unwrap()
+    }
+
+    pub fn max() -> Self {
+        Self::new(RAV1D_MAX_SEGMENTS - 1).unwrap()
+    }
```

Comment:
I removed the `const` since `.unwrap()` isn't stably `const` yet.  As for why go through `Self::new`, it's just to have only one entry point to construction that needs to be safety checked.  I could construct these directly here, but it's just slightly more mental overhead, and everything should be optimized out anyways, so I didn't see a need.